jsonschema-core.xml

```diff
@@ -189,7 +189,7 @@
                 <t>
                     A JSON Schema document, or simply a schema, is a JSON document used to describe an instance.
                     A schema is itself interpreted as an instance.
-                    A JSON schema MUST be an object.
+                    A JSON Schema document MUST consist of an object, known as the root schema.
                 </t>
```

Comment:
I think it's fixed in #167, which currently has more support than this PR.  If that changes I will backport @awwright's language fixes to this PR.